lib/money-rails/test_helpers.rb

```diff
@@ -95,6 +101,14 @@ def test_currency_iso
         end
       end
 
+      def test_currency_attribute
+        if @currency_attribute
+          @actual.send(@money_attribute).currency == @actual.send(@currency_attribute)
```

Comment:
Can you please change `send`s to `public_send`s?

Reply:
I've replaced all instances of `@actual.send` with `@actual.pubic_send`. Is that what you wanted, @antstorm? What is the reasoning behind this?